src/bundles/files.js

```diff
@@ -390,6 +378,11 @@ export default (opts = {}) => {
 
     selectFilesIsFetching: (state) => state.files.pending.some(a => a.type === actions.FETCH),
 
+    selectShowLoadingAnimation: (state) => {
+      const pending = state.files.pending.find(a => a.type === actions.FETCH)
+      return pending ? (Date.now() - pending.start) > 1000 : false
```

Comment:
of note, this selector will only be re-run if something happens, like an `APP_IDLE` event, or the user interacts with the app. That's why it will sometimes take a while to see the loading animaton... We're going to revisit this in a future PR so no need to change it now.